src/pem_common_plug.c

```diff
@@ -51,6 +52,19 @@ int pem_valid(char *ciphertext, struct fmt_main *self)
 	ctcopy += TAG_LENGTH;
 	if ((p = strtokm(ctcopy, "$")) == NULL) // type
 		goto err;
+	if (*p != '1' && !warned) {
```

Comment:
This allows any strings starting with `1` (and treats them the same). I suggest we use `strcmp(p, "1")` here.